.github/workflows/root-ci-config/build_root.py

```diff
@@ -335,5 +338,28 @@ def rebase(base_ref, head_ref, shell_log) -> str:
     return shell_log
 
 
+@github_log_group("Create Test Coverage in XML")
+def create_coverage_xml(shell_log: str) -> str:
+    result, shell_log = subprocess_with_log(f"""
+        pwd
+        cd '{WORKDIR}/build'
+        gcovr --cobertura-pretty --gcov-ignore-errors=no_working_dir_found --merge-mode-functions=merge-use-line-min -r ../src ../build -o cobertura-cov.xml
+        echo "This is test file" >> cobertura-cov.xml
```

Comment:
This seems to not be a good idea?! It will also change the exit code, i.e. if `gcovr` fails we won't notice because the exit code will be that of the subsequent `echo`.